File: median.py
def selectionSort(arr):

    origCopy = arr.copy()
    sorted_numbers = []

    def getMin(lt):
        # Assume the first number is the highest
        min = lt[0]
        for item in lt:
            if item < min:
                min = item
        return min
    
    while (len(origCopy) > 0):
        minNum = getMin(origCopy)
        sorted_numbers.append(minNum)
        origCopy.remove(minNum)

    return sorted_numbers

numbers = input("Enter numbers (Separated by input): ")
num_list = [int(num) for num in numbers.split(",")]

# Sort first
sorted_list = selectionSort(num_list)

# Get the middle
if (len(sorted_list) % 2 == 0):
    mid1 = len(sorted_list) // 2 - 1
    mid2 = len(sorted_list) // 2
    middle = [sorted_list[mid1], sorted_list[mid2]]
else:
    middle = [sorted_list[len(sorted_list) // 2]]

print(f"he median is/are: {", ".join(map(str, middle))}")

File: median.js
const input = require("./input");

function selectionSort(arr) {

    // Make original copy
    const origCopy = [...arr];
    const sortedArray = [];

    function geMin(arr) {
        // Assume the first number is the highest:
        let maxNum = arr[0];
        arr.forEach(item => {
            if (item < maxNum) maxNum = item;
        });
        return maxNum;
    }

    while (origCopy.length > 0) {
        const smallestNumber = geMin(origCopy);
        sortedArray.push(smallestNumber);
        const index = origCopy.indexOf(smallestNumber);
        if (index > -1) origCopy.splice(index, 1);
    }

    return sortedArray;
}

(async() => {
    
    const numbers = await input("Enter numbers (Separated by Comma): ");
    const num_list = numbers.split(",").map(Number);

    // Sort first
    const sorted_list = selectionSort(num_list);

    // Get the middle
    let middle;
    if (sorted_list.length % 2 === 0) {
        const mid1 = Math.floor(sorted_list.length / 2 - 1);
        const mid2 = Math.floor(sorted_list.length / 2);
        middle = [sorted_list[mid1], sorted_list[mid2]];
    } else {
        const mid = Math.floor(sorted_list.length / 2);
        middle = [sorted_list[mid]];
    }

    console.log(`The median is/are: ${middle.join(", ")}`);

})();